lib/Minz/Migrator.php

```diff
@@ -55,7 +55,7 @@ public static function execute($migrations_path, $applied_migrations_path) {
 		}
 
 		$lock_path = $applied_migrations_path . '.lock';
-		if (!@mkdir($lock_path, 0770, true)) {
+		if (!mkdir($lock_path, 0770, true) && !is_dir($lock_path)) {
```

Comment:
Why did you remove the `@`? It risks breaking some outputs

Reply:
Why the `!is_dir($lock_path)` ? This looks wrong

Reply:
that's for remove the @

Reply:
But I do not believe your new version is equivalent at all. The first mkdir is supposed to fail if the path exists. It is a mutex

Reply:
I am reverting this line, as i believe the new one is wrong

```suggestion
		if (!@mkdir($lock_path, 0770, true)) {
```